base/libgit2/libgit2.jl

```diff
@@ -132,11 +149,23 @@ function diff_files(repo::GitRepo, branch1::AbstractString, branch2::AbstractStr
     return files
 end
 
+"""
+    is_ancestor_of(a::AbstractString, b::AbstractString, repo::GitRepo) -> Bool
+
+Returns `true` if `a`, a [`GitHash`](@ref) in string form, is an ancestor of
+`b`, a [`GitHash`](@ref) in string form.
```

Comment:
Probably an unnecessary change, but I think this would read slightly better as
> Return `true` if `a` is an ancestor of `b`, where `a` and `b` are ``[`GitHash`](@ref)``es in string form.